class/defaults.yml

```diff
@@ -8,13 +8,13 @@ parameters:
     charts:
       topolvm:
         source: https://topolvm.github.io/topolvm
-        version: v5.0.0
+        version: v6.0.0
 
     images:
       topolvm:
-        registry: quay.io
+        registry: ghcr.io
```

Comment:
Changing the image source should be mentioned in the upgrade doc. https://github.com/projectsyn/component-topolvm/blob/master/docs/modules/ROOT/pages/how-tos/upgrade-v1-to-v2.adoc Presuming that there might be firewall rules blocking ghcr.io.

Reply:
good point, will make a version 3 in that case.